test/metabase/driver/postgres_test.clj

```diff
@@ -7,9 +7,7 @@
              [query-processor-test :refer [rows]]
              [sync :as sync]
              [util :as u]]
-            [metabase.driver
-             [generic-sql :as sql]
-             postgres]
```

Comment:
Yeah, I've been having issues with that, running cljr-clean-ns, then it breaking the code. I'll add this back.